### DISCUSSION TITLE
The msi log file status does not reflect the installation files status

### DISCUSSION THREAD

**Discussion by Diwakar243** [wixtoolset/issues#6622] [Questions] [answered]
Trying to upgrade the msi in which a component has its ID,GUID and FileID are changed using harvesting.
I am expecting WiX to treat this as a new component and install it straight away. The log also shows states the same

MSI (s) (00:14) [21:23:39:980]: Component: NLog.config; Installed: Absent;   Request: Local;   Action: Local

As the Installed State is absent and the Request and Action are local, the component should be installed.
But NLog.config is not getting installed in our case.

Did anyone else face the same issue or have a solution for this?

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by robmen** [comment, marked as answer]
New GUID means the Component isn't installed. Try reading: https://robmensching.com/blog/posts/2003/10/18/component-rules-101/
